introduction.md

```diff
@@ -0,0 +1 @@
+Hector Js.Working to achieve the badges and see what new thing I learn from codeschool. FreeWeekend
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.